src/plugins/ReviewEnforcer/review_enforcer.ts

```diff
@@ -67,12 +67,9 @@ const checkPythonPRFiles = async (context: PRContext) => {
 
 const markForReview = async (context: PRContext) => {
   await Promise.all([
```

Comment:
If we're only doing 1 thing, we don't need to `Promise.all` it (that's like `asyncio.gather`)